hack/delete

```diff
@@ -19,7 +19,7 @@ NAME="$2"
 NAMESPACE=""
 
 if [[ "$1" != "project" ]]; then
-  NAMESPACE="--namespace ${3:-$(id -u -n)}"
+  NAMESPACE="--namespace ${3:-"garden-dev"}"
```

Comment:
I'm not sure whether I want to have this as a default behavior... 
I'd rather vote for removing this change.